core/storage/btree.rs

```diff
@@ -1555,23 +1552,16 @@ fn find_free_cell(page_ref: &PageContent, db_header: Ref<DatabaseHeader>, amount
 
 impl Cursor for BTreeCursor {
     fn seek_to_last(&mut self) -> Result<CursorResult<()>> {
-        self.move_to_rightmost()?;
-        match self.get_next_record(None)? {
-            CursorResult::Ok((rowid, next)) => {
-                if rowid.is_none() {
-                    match self.is_empty_table()? {
-                        CursorResult::Ok(is_empty) => {
-                            assert!(is_empty)
-                        }
-                        CursorResult::IO => (),
```

Comment:
@pereman2 and this, should it return or not? when the instruction restarts it will advance the cursor again in `get_next_record()` - does it matter because the next time it's going to be `None` again anyway...?

Reply:
looks like it should return io, not mega sure but looks like it